lib/cylc/task_message.py

```diff
@@ -235,6 +238,12 @@ def _update_job_status_file(self, messages):
                     job_status_file.write("%s=%s\n" % (
                         self.CYLC_JOB_EXIT,
                         message.replace(self.FAIL_MESSAGE_PREFIX, "")))
+                elif message.startswith(self.ABORT_MESSAGE_PREFIX):
+                    job_status_file.write("%s=%s\n" % (
+                        self.CYLC_JOB_EXIT, "EXIT"))
+                    job_status_file.write("%s=%s\n" % (
+                        self.CYLC_JOB_EXIT_MESSAGE,
+                        message.replace(self.ABORT_MESSAGE_PREFIX, "")))
```

Comment:
I think you can just set `CYLC_JOB_EXIT` to the abort message. If I remember correctly, it can be anything other than `SUCCEEDED`, `ERR` and `EXIT`. `cylc jobs-poll` uses it as the value of the *signal* which can be any single-line strings.

Otherwise, if we want to introduce `CYLC_JOB_EXIT_MESSAGE`, I suppose `CYLC_JOB_EXIT` should be set to `ABORT` or `ABRT` (or something like that), and you may want to ensure that `cylc jobs-poll` can return it.